### PR TITLE
Fix/babo/nav bar cleanup

### PR DESCRIPTION
**Pull request by nibab-boo**
- [x] Fixed image on Banner
- [x] Nested selection for Nav-link
- [x] Fixed Icon fill for Top & Bottom. But, there is a small issue for Project. Icon looks smaller. I would be helpful if you could share a way to fix it.
- [x] removed unnecessary import and fixed margin for bottom nav

### CODE REVIEW

File: layout/GeneralLayout/index.tsx [hunk 1]
```diff
@@ -24,22 +16,30 @@ import styles from './index.module.scss';
...
     title: 'Your Projects',
     subTitle: 'Manage and create projects',
-    img: 'home-image',
+    img: projectImg,
   },
   network: {
     title: 'Network',
     subTitle: 'Connect with skilled professionals',
-    img: 'home-image',
+    img: projectImg,
```
Comment (lalomartins) [inline layout/GeneralLayout/index.tsx:42]: network

**Review by lalomartins — CHANGES_REQUESTED** [review]
looks good otherwise 🙏

File: layout/GeneralLayout/index.tsx (hunk 1 above)
Comment (nibab-boo) [reply layout/GeneralLayout/index.tsx:42]: We still don't have the network img.

Comment (lalomartins) [reply layout/GeneralLayout/index.tsx:42]: ah yes, yesterday I thought this was the icon. I guess it's good enough for now then

**Review by lalomartins — APPROVED** [review]

**Pull request merged**